services/web/server/src/simcore_service_webserver/studies_dispatcher/module_setup.py

```diff
@@ -40,3 +40,6 @@ def setup_studies_dispatcher(app: web.Application) -> bool:
         strict=True,
     )
     app.router.add_routes(rest_routes)
+
+    # TODO: add a background task that builds _FILETYPE_TO_VIEWER with viewers
+    #       shared with everybody.
```

Comment:
are all viewers going to be shared with everybody?

Reply:
I do not expect so. This is a note to implement in next PR since I needed your input